Azure/azure-cli
# {Graph} az aro module migration to Microsoft Graph

**Related command**
az aro

**Description**
Migrating from Azure SDK's AAD Graph to Microsoft Graph 
#22174 

**Testing Guide**
All the commands were executed successfully.


---

This checklist is used to make sure that common guidelines for a pull request are followed.
